javascript/workflow/src/verbs/convert.ts

```diff
@@ -17,13 +17,14 @@ import { stepVerbFactory } from './util/factories.js'
  */
 export const convertStep: ColumnTableStep<ConvertArgs> = (
 	input,
-	{ columns, type, radix, delimiter, formatPattern },
+	{ column, type, radix, delimiter, formatPattern },
 ) => {
-	// note that this applies the specified parse to every column equally
-	const dArgs = columns.reduce((acc, cur) => {
+	const dArgs = column.split(' ').reduce((acc, cur) => {
```

Comment:
You shouldn't need the column.split anymore - it should just be a single column name with no delimiter, so you can remove the reducer entirely